docs/source/index.rst

```diff
@@ -11,13 +11,13 @@
 Welcome to Kedro's documentation!
```

Comment:
```suggestion
Welcome to Kedro's award-winning documentation!
```

Reply:
While you're in there making changes, are you OK to commit this as part of the set of changes?

Reply:
Absolutely!